src/components/CustomizableAvatar.js

```diff
@@ -0,0 +1,17 @@
+import Image from 'next/image'
+function CustomizableAvatar({
```

Comment:
Can you add a screenshot of the UI from localhost, for this component?

Reply:
Added in! @jjleigh